transmogrifier/sources/transformer.py

```diff
@@ -627,7 +625,7 @@ def get_source_link(
 
     @classmethod
     def get_timdex_record_id(
-        cls, source: str, source_record_id: str, source_record: Tag
+        cls, source: str, source_record_id: str, source_record: Tag  # noqa: ARG003
```

Comment:
I landed on this one for a comment, but I think it's applicable to any of the methods that have an argument that is defined on the parent class, but not used in the child classes.  However, only works when the argument is **positional**; don't think it'd be as simple for named arguments.

We could also lean on the leading underscore to indicate the argument is not used and thereby avoid linting ignores:

```python
cls, source: str, source_record_id: str, _source_record: Tag
```

This is not a requested change, just noting.  This would likely require multiple changes, and I'm not sure if it actually improves readability.  Although, if one found themselves updating the codebase and wanted to use the variable later, they might pause with the leading underscore and think to themselves it wasn't used previously.  

All in all, probably not worth the change, but wanted to note.

Reply:
A little bit of work but necessary, fixed!